src/main/webapp/createYourOwn.html

```diff
@@ -22,6 +22,51 @@
             </nav>
         </div>
       </header>
+      <h1>New Initiative</h1>
+      <hr style="height:4px;background-color:#F8944C;border-width:0px;color:#F8944C;width:300px;margin-bottom:30px">
+      <div id="new-initiative">
+          <form id="newInitiativeForm" action="myUploadUrl" method="POST" enctype="multipart/form-data">
+            <p class="initiative-input">Title:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
```

Comment:
Since all images seem to have the same style, why not consider including the styling in the same CSS file?